compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -1031,125 +1028,131 @@ public BLangNode transform(ObjectFieldNode objFieldNode) {
+//            bLService.annAttachments = applyAll(getAnnotations(serviceDeclrNode.metadata()));
+//        }
+//
+//        // We add all service nodes to top level, only for future reference.
+//        addToTop(bLService);
+//
+//        // 1) Define type nodeDefinition for service type.
+//        BLangClassDefinition classDef = (BLangClassDefinition) TreeBuilder.createClassDefNode();
+//        BLangIdentifier serviceTypeID = createIdentifier(identifierPos, serviceTypeName);
+//        serviceTypeID.pos = pos;
+//        classDef.setName(serviceTypeID);
+//        classDef.flagSet.add(SERVICE);
+//
+//        if (!isAnonServiceValue) {
+//            addServiceConstructsToClassDefinition((ServiceBodyNode) serviceDeclrNode.serviceBody(), classDef);
+//            bLService.markdownDocumentationAttachment =
+//                    createMarkdownDocumentationAttachment(getDocumentationString(serviceDeclrNode.metadata()));
+//        } else {
+//            serviceConstructorNode = (ServiceConstructorExpressionNode) serviceNode;
+//            addServiceConstructsToClassDefinition((ServiceBodyNode) serviceConstructorNode.serviceBody(), classDef);
+//            bLService.annAttachments = applyAll(serviceConstructorNode.annotations());
+//        }
+//
+//        classDef.pos = pos;
+//        addToTop(classDef);
+//        bLService.serviceClass = classDef;
+//
+//        // 2) Create service constructor.
+//        final BLangServiceConstructorExpr serviceConstNode = (BLangServiceConstructorExpr) TreeBuilder
+//                .createServiceConstructorNode();
+//        serviceConstNode.serviceNode = bLService;
+//        serviceConstNode.pos = pos;
+//
+//        // Crate Global variable for service.
+//        bLService.pos = pos;
+//        if (!isAnonServiceValue) {
+//            BLangSimpleVariable var = (BLangSimpleVariable) createBasicVarNodeWithoutType(identifierPos,
+//                    Collections.emptySet(),
+//                    serviceName, identifierPos,
+//                    serviceConstNode);
+//            var.flagSet.add(Flag.FINAL);
+//            var.flagSet.add(SERVICE);
+//
+//            BLangUserDefinedType bLUserDefinedType = (BLangUserDefinedType) TreeBuilder.createUserDefinedTypeNode();
+//            bLUserDefinedType.pkgAlias = (BLangIdentifier) TreeBuilder.createIdentifierNode();
+//            bLUserDefinedType.typeName = classDef.name;
+//            bLUserDefinedType.pos = pos;
+//
+//            var.typeNode = bLUserDefinedType;
+//            bLService.variableNode = var;
+//            return var;
+//        } else {
+//            BLangServiceConstructorExpr serviceConstructorExpr =
+//                    (BLangServiceConstructorExpr) TreeBuilder.createServiceConstructorNode();
+//            serviceConstructorExpr.serviceNode = bLService;
+//            return serviceConstructorExpr;
+//        }
+//    }
+
     public void addServiceConstructsToClassDefinition(ServiceBodyNode serviceBodyNode,
```

Comment:
this looks like dead code, if so shall we remove it

Reply:
yea. will remove. thanks